modules/tinymce/src/core/main/ts/api/OptionTypes.ts

```diff
@@ -117,6 +117,7 @@ interface BaseEditorOptions {
   font_size_style_values?: string;
   font_size_formats?: string;
   font_size_input_default_unit?: string;
+  force_hex_color?: 'always' | 'rgb_only' | 'off';
```

Comment:
Probably worth having an exported type at the top of the file for these values. e.g. `export type ForceHexColor = 'always' | 'rgb_only' | 'off'`

That way it can be shared and possible used in the Options.ts file as well

Reply:
Added a ForceHexColor export to OptionTypes.ts